nixos/modules/services/databases/openldap.nix

```diff
@@ -54,6 +54,12 @@ in
         description = "The database directory.";
       };
 
+      logLevel = mkOption {
+        type = types.int;
```

Comment:
Can you make this `types.string`, and add a link to https://www.openldap.org/doc/admin24/slapdconfig.html#loglevel%20%3Clevel%3E in the description?

Being able to set `logLevel = "acl trace";` seems a bit more clear than `129`.